index.html

```diff
@@ -134,30 +135,33 @@ <h3>Frameworks</h3>
             <h3>Skills</h3>
           </div>
           <ul id="skills-list">
-            <li>Git</li>
-            <li>Github</li>
-            <li>Terminal</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Codekit</li>
-          </ul>
+            <li>Database Management</li>
+            <li>Version Control</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Web Development</li>
+           </ul>
         </div>
       </div>
     </section>
     <!--Form section-->
     <section id="form-container">
       <h2>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
+        I'm always interested <br> in hearing about new <br> projects, so if you'd <br> like to chat please <br> get in touch.
       </h2>
       <form action="https://formspree.io/f/mzbowdel" method="POST" id="contact-form">
-        <fieldset> <input class="inputs" type="text" name="full-name" id="fullname" maxlength="30" placeholder="Enter your name" required/>
-          <input class="inputs" type="email" name="email" id="email" placeholder="Email" required/>
-          <textarea name="comments" id="comments" maxlength="500" placeholder="Enter text here"> </textarea>
+        <input class="input1" type="text" name= full-name" id="fullname" maxlength="30" placeholder="Full name"  required/> 
+      
+        <input class="input2" type="text" id="lastname" name="lastname" maxlength="30" placeholder="Last Name" required>
+        <input class="inputs" type="email" name="email" id="email" placeholder="Email adress" required/> 
+        <fieldset>
+          <textarea name="comments" id="comments" maxlength="500" placeholder="Write me something..."> </textarea>
         </fieldset>
         <input type="submit" id="submit" name="submit" value="Get in touch" />
       </form>
     </section>
```

Comment:
Good job placing the paragraph beside the form as suggested by the previous reviewer. 👍🏿 However, you would need to make some slight adjustments to it to make it look close to the FIgma design. 🎨 

- [ ] Please check and reduce the space between the paragraph and the form as seen in the Figma design. ↔️ 

- [ ] Please ensure that the social media icons are centered at the bottom of the page as seen in the Figma design. ✔️ 

- [ ] Kindly make sure that you have a placeholder for the `Textarea` input field of the form that says **"Write me something..."** as seen in the Figma design. 📜 

### Your design ⬇️ 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/117144028/4bba01d0-4177-45c0-8b0e-bf5348e9513f)

### Figma design ⬇️
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/117144028/3611739b-55d7-450b-8b56-82d8c782bbfa)